### PR TITLE
Adjust input paths for dependencies which reject slashes on Windows

### PR DESCRIPTION
**Pull request by uckelman-sf**

### CODE REVIEW

**Comment by simsong** [comment]
Appveyor has suddenly started working fast and reliably 😌

**Comment by uckelman-sf** [comment]
That's strange. Nothing in this PR should have affected AppVeyor.

**Comment by simsong** [comment]
I think that AppVeyor is just inconsistent. And it is somewhat different
than the GitHub Actions we are using for testing.

**Comment by uckelman-sf** [comment]
@simsong Ready for review.

File: test/fiwalk/fiwalk_test.cpp [hunk 1]
```diff
@@ -15,9 +15,6 @@
 #include "tools/fiwalk/src/fiwalk.h"
 
 void check_image(std::string img_path, std::string dfxml2_path) {
-#if defined(__MINGW32__) || defined(__MINGW64__)
-    WARN("fiwalk_test disabled under mingw. Will not test "+img_path);
-#else
```
Comment (simsong) [inline test/fiwalk/fiwalk_test.cpp:20]: yay!

**Review by simsong — APPROVED** [review]

**Pull request merged**